callisto/notification/api.py

```diff
@@ -0,0 +1,95 @@
+import logging
+
+import gnupg
+import pytz
+
+from django.conf import settings
+from django.core.mail.message import EmailMultiAlternatives
+from django.utils import timezone
+
+from callisto.delivery.models import SentMatchReport
+from callisto.notification.models import EmailNotification
+
+logger = logging.getLogger(__name__)
+tzname = settings.REPORT_TIME_ZONE or 'America/Los_Angeles'
+timezone.activate(pytz.timezone(tzname))
+
+
+class NotificationApi(object):
```

Comment:
I think what I'd like to see is an abstract NotificationApi interface in *delivery* and the implementation in this app. the default notifier could be set in settings (I'm not 100% sure on how this would work--a string name with dynamic instantiation?), and then an implementer could potentially override that setting and avoid including the notification app at all. for now the implementation could look exactly like this file, but eventually you could take a similar approach with PDF generation and separate out those bits into an abstract API at point of call and an overridable implementation elsewhere to address #150